website-challenge-VicRenRen/styles.css

```
@@ -0,0 +1,205 @@
+.navbar {
+    overflow: hidden;
+    background-image: url("images-merlin-jones-website/merlin-navbar-background.png");
+    background-size: cover;
+    position: fixed; /* Set the navbar to fixed position */
+    z-index:10;
+    top: 0; /* Position the navbar at the top of the page */
+    right: 0;
+    width: 100%; 
+  }
+  
+  /* Links inside the navbar */
+.navbar a {
+    font-family: "Courier New", monospace;
+    float: left;
+    display: block;
+    color: greenyellow;
+    text-align: right;
+    padding: 14px 16px;
+    text-decoration: none;
+}
+
+.navbar a:hover {
+    background: green;
+    color: whitesmoke;
+}
+
+/* all images */
+.image {
+    display: block;
+    width: 100%;
+    height: auto;
+}
+
+/* Home section */
+
+.home-text {
+    font-family: "Courier New", monospace;
+    letter-spacing: 5px;
+    text-align: center;
+    color: greenyellow;
+    font-size: 70px;
+    position: absolute;
+    top: 50%;
+    left: 25%;
+    transform: translate(-50%, -50%);
+    -ms-transform: translate(-50%, -50%);
+}
+
+/* About section */
+.about {
+    position: relative;
+    width: 100%;
+    height: auto;
+}
+.about-text {
+    font-family: "Courier New", monospace;
+    letter-spacing: 5px;
+    text-align: center;
+    color: black;
+    font-size: 50px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    -ms-transform: translate(-50%, -50%);
+}
+.about-image {
+position: absolute;
+display: block;
+border-radius: 15px 50px 30px;
+}
```

It's better to have code formatted consistently, so install a linter such as Prettier to work towards a clean style.

```suggestion
.about-image {
    position: absolute;
    display: block;
    border-radius: 15px 50px 30px;
}
```